tests/angular/package.json

```diff
@@ -1,20 +1,45 @@
 {
   "name": "@ni/angular-test",
   "version": "1.0.0",
-  "private": true,
   "description": "Smoke test for Angular project integration.",
-  "type": "module",
   "scripts": {
-    "test": "eslint ."
-  },
-  "peerDependencies": {
-    "@ni/eslint-config-angular": "*",
-    "@ni/eslint-config-javascript": "*"
+    "start": "ng serve",
+    "build": "ng build && ng build angular-library",
+    "lint": "ng lint",
+    "ng-test": "ng test --watch=false",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng lint && eslint . && cd projects/angular-library && eslint ."
```

Comment:
maybe we should also go ahead and run `ng-test` as part of `test` so it gets exercised.